pkg/parser/loa/parse.go

```diff
@@ -0,0 +1,221 @@
+package loa
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/transcom/mymove/pkg/models"
+)
+
+const timeConversionMethod = "2006-01-02 15:04:05"
```

Comment:
Where does this specific time and date come from?

Reply:
Mimicking from here: https://go.dev/src/time/format.go